model/src/pyrenew/latent/infections.py

```diff
@@ -49,14 +49,14 @@ class Infections(RandomVariable):
 
     def __init__(
         self,
-        infections_mean_varname: str = "latent_infections",
+        infections_varname: str = "latent_infections",
```

Comment:
```suggestion
        infections_varname: str,
```

As per [DHM](https://github.com/CDCgov/multisignal-epi-inference/pull/205#pullrequestreview-2131069672)'s comment.